docs/hardware/hardware-selection.md

```diff
@@ -0,0 +1,278 @@
+      * BetaFPV SuperD (20dBm)
+
+??? info "Size"
+
+    The FPV world shook when ELRS released receivers that were half the size of "nano" sized receivers, included the antenna onboard, and still had kilometers of range at 250Hz/100mW. A small receiver can fit in tight places, but remember that tucking a tiny receiver's ceramic antenna deep inside a stack behind carbon reduces its performance, which was already compromised by the elimination of amplifiers to make it that small. Larger ELRS recievers regain these amplifiers offering better reception and telemetry range at the cost of size and weight. The common size classes are shown below, but there are other recievers with slightly different sizes:
+
+    <figure>
+      <img class="center-img" src="../../assets/images/hw-selection/sizes.png"/>
+    </figure>
+
+    PP Sized Recievers (absolute smallest, lowest range regardless of antennas):
+
+      * Happymodel EP/PP
+      * Radiomaster RP
+      * BetaFPV Lite
+
+    Nano Sized Recievers (medium sized, can have PA/LNA but tends to lack some features):
+
+      * BetaFPV Nano
+      * iFlight RXes
+      * Namimno Flash Diversity RX
+      * Axisflying Thor RX
+      * Namimno Flash RX
+
+    Larger Recievers (most feature rich but also significantly larger)
+
+      * Matek R24-D
+      * Radiomaster RP3
+      * BetaFPV SuperD
+
+## Transmitter Selection
+This selection lists out some common features of transmitters and some examples of transmitters with those features
+
+Everyone has different needs for a transmitter but the main needs come down to size, range and integration.
+
+??? info "Built-In Transmitters"
+
+    Certain manufacturers have made handsets with built in ELRS modules, which offer tigher integration with ELRS. These can be update through the handset as well as generally over WiFi as is standard.
+
+    Some handsets with built in ELRS include:
+
+      * RadioMaster TX16S ELRS
+      * RadioMaster Zorro ELRS
+      * BetaFPV Lite Radio 3 Pro ELRS
+      * Jumper T-Pro
+      * Jumper T-Lite v2
+
+    A notable exception to this list is the iFlight Commando, which, while including a TX built into the handset is simply wired as an external module, and put within the casing. It does support an external module and notably has both 868/915MHz and 2.4GHz options available
+
+??? info "Frequency"
+
+    ExpressLRS offers both 2.4GHz and 900MHz systems, with each only working with receivers and transmitters of the same frequency. 2.4GHz is the currently most popular frequency given it's legality, featureset and cost. 2.4GHz links offer the fastest packet rates allowing for a more locked in feel while piloting and still offer massive amounts of range. 900MHz is the original long range frequency and still can be great for high wifi pollution environments with slightly better penetration.
+
+    For new users unless you plan on flying hundreds of kilometers or in a high noise environment we reccomend 2.4GHz hardware:
```

Comment:
Recommend is misspelled here

this section sounds like these are the ONLY ones we  recommend. Use "some of our recommendations" instead or any wording like that. ;)